src/sentry/services/hybrid_cloud/integration/__init__.py

```diff
@@ -125,7 +125,7 @@ def get_integrations(
         limit: int | None = 5,
     ) -> List[APIIntegration]:
         """
-        Returns all APIIntegrations matching the provided kwargs. Defaults to a limit of 5.
+        Returns all APIIntegrations matching the provided kwargs.
```

Comment:
The default values is still in this method prototype. Should it be removed?

Reply:
Good catch!